### PR TITLE
[FrameworkBundle] Update cache:clear help

### PR DESCRIPTION
**Pull request by BafS**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | yes ? (kind-of)
| New feature?  | no
| Deprecations? | no
| Tickets       | no
| License       | MIT

The current help is a bit confusing and does not match with what the command do (`Clear and Warmup the cache.`).

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php [hunk 1]
```diff
@@ -59,7 +59,7 @@ protected function configure()
             ])
             ->setDescription('Clear the cache')
             ->setHelp(<<<'EOF'
-The <info>%command.name%</info> command clears the application cache for a given environment
+The <info>%command.name%</info> command clears and warmup the application cache for a given environment
```
Comment (Tobion) [inline src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php]: lang: and warms up

Comment (BafS) [reply src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php]: updated

Comment (BafS) [reply src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php]: Shall we change the class doc too ? Currently we have `Clear and Warmup the cache.`

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @BafS.

**Pull request merged**